README.md

```diff
@@ -115,56 +121,67 @@ The configuration file is a simple json document, a sample can be seen bellow:
-    * counterAggregations [optional] [default: [Aggregation.COUNT, Aggregation.SUM] ] - aggregations to be applied to counter based metrics
-    * counterFrequency [optional] [default: AggregationFreq.FREQ_10] - aggregation frequency for counter based metrics
-    * gaugeAggregations [optional] [default: [Aggregation.LAST, Aggregation.MAX, Aggregation.AVG] ] - aggregations to be applied to gauge based metrics
-    * gaugeFrequency [optional] [default: AggregationFreq.FREQ_10] - aggregation frequency for gauge based metrics
-    * http-server-url-patterns [optional] - patterns to transform urls for metrics collection
-    * http-server-ignore-url-patterns [optional] - patterns of urls that you won't want tracked
-    * gauge-collection-interval [optional] [default: ] - to avoid reporting gauges every time it changes
-    * httpMetricsPath [optional] [default: '/tel/v2.0/metrics'] - path to send metrics to when http transport is set
-    * transport [optional] [default: HTTP] - type of transport to be used when sending metrics to statful (UDP/HTTP)
+## Configurations
+The following section presents detailed information on the available options for setup in the configuration parameters.
+
+### Global Configurations
+
+| Option | Description | Type | Default | Required |
+|:---|:---|:---|:---|:---|
+| _host_ | Defines the hostname to where the metrics are sent. | `string` | `api.statful.com` | **NO** |
+| _port_ | Defines the port to where the metrics are sent. | `string` | `443` | **NO** |
+| _secure_ | Enables or disables https. <br><br>Not yet supported. | `boolean` | `true` | **NO** |
+| _timeout_ | Defines the timeout for http transport, in **milliseconds**.<br><br>Not yet supported. | `number` | `2000` | **NO** |
+| _token_ | Defines the token used to match incoming data to Statful.| `string` | **none** | **YES** |
+| _app_ | Defines the application's global name. When specified, it sets a global tag like `app=setValue`. | `string` | **none** | **NO** |
+| _dryRun_ | Defines if metrics should be output to the logger instead of being sent to Statful (useful for testing/debugging purposes). | `boolean` | `false` | **NO** |
+| _tags_ | Object for setting the global tags. | `object` | `{}` | **NO** |
+| _sampleRate_ | Defines the rate sampling. <br><br>**It should be a number between [1, 100]**. | `number` | `100` | **NO** |
+| _namespace_ | Defines the global namespace. A prefix could be set if the user sends metrics through Statful. | `string` | `application` | **NO** |
+| _flushSize_ | Defines the maximum buffer size before performing a flush. | `number` | `10` | **NO** |
+| _flushInterval_ | Defines an interval to periodically flush the buffer based on time. | `number` | `30000` | **NO** |
+| _maxBufferSize_ | Defines how many metrics at max are kept in the buffer between forced flushes. | `number` | `5000` | **NO** |
+
+
+### Vert.x Specific Configurations
```

Comment:
This kind of makes it seem like they are Vert.x configs, when they are just Statful configs specific to this client

Reply:
**Statful's Configurations Specific to Vert.x**?